Example/Tests/GINISupportedFormatsViewControllerTests.swift

```diff
@@ -27,22 +27,14 @@ final class GINISupportedFormatsViewControllerTests: XCTestCase {
         _ = supportedFormatsViewController.view
     }
     
-    func testSectionsCount() {
+    func testSectionsMockedCount() {
```

Comment:
You aren ot testing the mocked count, but the count in general. I'd remove the "Mocked" part.